pybossa/jobs.py

```diff
@@ -1297,6 +1319,41 @@ def get_management_dashboard_stats(user_email):
     send_mail(mail_dict)
 
 
+def check_and_send_project_progress(project_id, conn=None):
+    from pybossa.cache.helpers import n_available_tasks
+    import pybossa.cache.users as cached_users
+    from pybossa.core import project_repo
+    from sqlalchemy.sql import text
```

Comment:
local imports?

Reply:
Local imports are used because they are not used in other functions, but I do not have preference using local imports over global imports, can follow conventions. So change to global imports?

Reply:
yes, global imports is fine.